manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "store-image",
-  "version": "0.5.0",
+  "version": "0.5.1",
```

Comment:
No need to update it manually. The release bot will do that for you!

Reply:
hmmm, thanks :)

Reply:
Don't forget to remove this change from your branch